analysis/detection/explorer/multiple_dimensions.py

```diff
@@ -8,15 +8,22 @@
 from analysis.detection.explorer.dimension_evaluator import DimensionEvaluator
 from analysis.detection.explorer.top_level import TopLevelEvaluator
 from analysis.configuration.configs import AnalysisProfile
+from analysis.configuration.processing_dates import ProcessingDateRange
 
 # TODO GLE only doing additive algm not the ratio algm.
 
 
 class MultiDimensionEvaluator(DimensionEvaluator):
-    def __init__(self, profile: AnalysisProfile, date_ranges: dict):
+    def __init__(
+        self,
+        profile: AnalysisProfile,
+        previous_date_range: ProcessingDateRange,
+        current_date_range: ProcessingDateRange,
+    ):
         # TODO GLE currently the profile only references percent_change.
         self.profile = profile
-        self.date_ranges = date_ranges
+        self.previous_date_range = previous_date_range
+        self.recent_date_range = current_date_range
```

Comment:
I wonder if we should be more consistent re recent_date_range and current_date range?
It will be easier to follow if we use on term to refer to the analysis period/results we are looking at.
Same for baseline vs previous.

Reply:
Yes, definitely.  I actually thought I caught all the discrepancies.  Thanks for the eagle eye on this.

Reply:
After chatting @wwyc  and I (@gleonard-m ) decided to standardize on baseline and current terminology.  

- `baseline_metric_value` & `current_metric_value`
- `baseline_period` & `current_period`